examples/ex34.hpp

```diff
@@ -0,0 +1,819 @@
+//                                        MFEM Example 34
+
+#pragma once
```

Comment:
I'd prefer to keep this. Is there a known issue with supported compilers?

Reply:
No, sorry this was out of habit. I am open to leaving or removing it